site/en/docs/extensions/migrating/mv2-sunset/index.md

```diff
@@ -10,94 +10,25 @@ tags:
   - extensions-news
 ---
 
-{% Aside %}
+As Manifest V3 approaches full feature parity with Manifest V2, we'll progressively phase out Manifest V2.
```

Comment:
I don’t think this sentence is clear. Does it mean phasing out individual features of v2? that all features in v2 are coming to v3?

Reply:
Agreed, this came from the original page but I think you're right. Removed that line and also linked out to known issues - let me know what you think 😄